File: scratch/EBM2x_legacy/RraSdc/model/BhfCustSaveReq.cs
using System;
using System.Collections.Generic;

namespace EBM2x.RraSdc.model
{
    public class BhfCustSaveReq {
	    public string tin { get; set; }  // TIN
	    public string bhfId { get; set; }  // Branch ID
	    public string custNo { get; set; }  // Customer Number
	    public string custTin { get; set; }  // Customer TIN
	    public string custNid { get; set; }  // Customer National ID
	    public string custNm { get; set; }  // Customer Name
	    public string custBhfId { get; set; }  // Customer Branch Office ID
	    public string adrs { get; set; }  // Address
	    public string telNo { get; set; }  // Contact
        public string email { get; set; }  // 이메일
        public string faxNo { get; set; }  // 팩스 번호
        public string useYn { get; set; }  // 사용 여부
        public string remark { get; set; }  //  비고

        public string regrId { get; set; }  //   등록자 아이디
        public string regrNm { get; set; }  //    등록장 명
        public string modrId { get; set; }  //   수정자 아이디
        public string modrNm { get; set; }  //    수정자 명

    }
}

File: app/src/main/java/com/kra/paypoint/data/remote/model/master/BhfCustSaveReq.kt
package com.kra.paypoint.data.remote.model.master

import com.google.gson.annotations.SerializedName

data class BhfCustSaveReq(
    @SerializedName("tin") val tin: String,
    @SerializedName("bhfId") val bhfId: String,
    @SerializedName("custNo") val custNo: String,
    @SerializedName("custTin") val custTin: String?,
    @SerializedName("custNid") val custNid: String?,
    @SerializedName("custNm") val custNm: String,
    @SerializedName("custBhfId") val custBhfId: String?,
    @SerializedName("adrs") val adrs: String?,
    @SerializedName("telNo") val telNo: String?,
    @SerializedName("email") val email: String?,
    @SerializedName("faxNo") val faxNo: String?,
    @SerializedName("useYn") val useYn: String,
    @SerializedName("remark") val remark: String?,
    @SerializedName("regrId") val regrId: String?,
    @SerializedName("regrNm") val regrNm: String?,
    @SerializedName("modrId") val modrId: String?,
    @SerializedName("modrNm") val modrNm: String?
)
